index.js

```diff
@@ -170,17 +170,17 @@ module.exports.processV2 = (output, origin) => {
 module.exports.processV3 = (output, origin, locales = []) => {
   const data = JSON.parse(output.toString().trim());
   const { automatic_captions, formats, subtitles } = data;
-  const { fragments: audio_fragments, url: audio_url, format_id: audio_format, abr: audio_bitrate, protocol: audio_protocol } = data;
```

Comment:
This is the audio track that yt-dlp auto selects for us. It's the best available opus track. 

If there's no opus track available, I dunno how yt-dlp picks the next best track.

Reply:
These could probably be comments in the code, if they're worth mentioning here.

Reply:
done